compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -330,6 +331,15 @@ final class Conversions(
         builder.setEvaluationTimeout(timeout.toSeconds)
       case Error.CanceledByRequest() =>
         builder.setCancelledByRequest(empty)
+      case Error.NoSuchTemplate(templateId, oPackageMeta) => {
+        val nstBuilder =
+          proto.ScenarioError.TemplateDoesNotExist.newBuilder
+            .setTemplateId(convertIdentifier(templateId))
+        oPackageMeta.foreach(packageMeta =>
+          nstBuilder.setPackageMetadata(mkPackageMetadata(packageMeta))
+        )
+        builder.setTemplateDoesNotExist(nstBuilder.build)
+      }
```

Comment:
This is more a question of taste, but I do not understand why haskellers always want to wrap scala code in braces, when the indentation --which is enforced by the CI -- clearly show the scope. Feel free to keep it if you want.  

```suggestion
      case Error.NoSuchTemplate(templateId, oPackageMeta) => 
        val nstBuilder =
          proto.ScenarioError.TemplateDoesNotExist.newBuilder
            .setTemplateId(convertIdentifier(templateId))
        oPackageMeta.foreach(packageMeta =>
          nstBuilder.setPackageMetadata(mkPackageMetadata(packageMeta))
        )
        builder.setTemplateDoesNotExist(nstBuilder.build)
```

Reply:
Oh! I had no idea this would even parse! I thought you always needed the curlys for multiple statements